sormas-api/src/main/java/de/symeda/sormas/api/infrastructure/InfrastructureHelper.java

```diff
@@ -73,4 +74,22 @@ public static Integer getProjectedPopulation(Integer population, Date collection
 	public static BigDecimal getCaseIncidence(int caseCount, int population, int divisor) {
 		return new BigDecimal(caseCount).divide(new BigDecimal((double) population / divisor), 2, RoundingMode.HALF_UP);
 	}
+
+	public static JurisdictionLevel getSuperordinateJurisdiction(JurisdictionLevel jurisdiction) {
```

Comment:
this method should already be present in JurisdictionHelper, not sure i understand why moving it here is better